pkg/kinflate/constants/constants.go

```diff
@@ -19,3 +19,8 @@ package constants
 // KubeManifestFileName is the Well-Known File Name for a kubernetes app manifest.
 const KubeManifestSuffix = ".yaml"
 const KubeManifestFileName = "Kube-manifest" + KubeManifestSuffix
+
+// Configmap behaviors
+const Create = "create"
+const Replace = "replace"
+const Merge = "merge"
```

Comment:
We should suffix or prefix the above with `Behavior` or `ConflictBehavior` term.

Reply:
I will add it. Thank you.